src/Immunisatie-0-1/Cert/XIS-Server/imm-serve-1-1.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://hl7.org/fhir/STU3/testscript.sch" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TestScript xmlns="http://hl7.org/fhir" xmlns:nts="http://nictiz.nl/xsl/testscript"
+    nts:scenario="server">
+    <id value="imm-serve-1-1"/>
+    <version value="r4-1.0.0"/>
+    <name value="Immunization - XIS Server - Scenario 0.1 - Serve Immunization resources of van Putten"/>
+    <description value="Scenario 1.1 - Serve Immunization resources of van Putten"/>
+    <nts:patientTokenFixture href="imm-Patient-01-token.xml"/>
+
+    <nts:include value="variable.immunization.medication.url" name="pharmaceuticalProduct-url" sequenceNumber="0"/>
```

Comment:
Vette constructie. Leuk gebruik van iif.

Ik zou hier wel een `nts:in-targets` op zetten om te voorkomen dat hij eruit klapt als die variabele helemaal niet gebruikt wordt verder. Of: je kunt hem gewoon binnen de `<test>` hieronder zetten, dan is het mooi gegroepeerd. Het NTS-proces regelt dat de `<variable>` in de output op de juiste plaats terecht komt.

Verder de vraag: hoe weet je zeker dat de goede Medication in Immunization[sequenceNumber] zit. Volgens mij kunnen er in theorie ook 100 Immunizations opgestuurd worden, waarvan de goede nummer 37 is. Of niet?

Hoe gaat deze constructie verder om met uuid-references en misschien zelfs contained resources? Die falen je iif, maar zijn vervolgens ook niet geschikt om een read op te doen.

Reply:
Net gerubberduckt: als deze variabele echt alleen in NoMedication gebruikt wordt, hebben daar mensen bewust een keuze gemaakt van 'ik ondersteun reads op Medication dus ik refereer met absolute of relatieve url' en dan werkt dit gewoon